src/list.js

```diff
@@ -293,6 +293,11 @@ Sk.builtin.list.prototype.sq$contains = function (item) {
     }
     return false;
 };
+
+Sk.builtin.list.prototype.__contains__ = function(self, item) {
```

Comment:
haha no worries. thats what these things are for right 😄 and I think if you'd look into what I do I soon be giving you beer instead